shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -590,7 +591,14 @@ private void configureStatusBarForFullscreenFlutterExperience() {
       Window window = getWindow();
       window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
       window.setStatusBarColor(0x40000000);
-      window.getDecorView().setSystemUiVisibility(PlatformPlugin.DEFAULT_SYSTEM_UI);
+      WindowCompat.setDecorFitsSystemWindows(window, false);
+      if (Build.VERSION.SDK_INT < 30) {
```

Comment:
does it need to also check for `Build.VERSION.SDK_INT >= 19`?

Reply:
No, because it works for SDK >= 16! You can see that in the source code at the bottom of [this page](https://androidx.de/androidx/core/view/WindowCompat.html).

Reply:
it looks like this caused a regression for SDK_INT >= 30. See b/223628114

Reply:
Ok, I'm investigating a fix for this to re-land it.